wordpress-mobile/gutenberg-mobile
# Change mediaId from String to int in bridge for media upload

This fixes the block validation error when uploading media on Android, and matches media handling in Gutenberg and WP media library.

To test:
1. using WPAndroid, try to upload a new image from the device, verify it uploads.
2. exit the editor and verify it uploads the Post
3. verify on the web, that the Post is shown and no validation errors occur.

@SergioEstevao as chatted, I guess you have this covered on the iOS side, but mentioning you here and asking for review just in case (not needed if the change is obvious to you). Thanks!
